tests/test-class-amp-meta-box.php

```diff
@@ -67,6 +67,7 @@ public function test_enqueue_admin_assets() {
 		// Test inline script boot.
 		$this->assertTrue( false !== stripos( wp_json_encode( $script_data ), 'ampPostMetaBox.boot(' ) );
 		unset( $GLOBALS['post'] );
+		unset( $GLOBALS['current_screen'] );
```

Comment:
This is needed to counteract `set_current_screen( 'post.php' )` above. This caused `is_admin()` to be true on later tests.

Reply:
FYI: Probably better to such things in `tearDown` to ensure they are run regardless of whether the tests fail in this function. If a test fails, then the vars wouldn't get cleared.